<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>313</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="acceptButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="settingsTabs">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="common_tab">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>10</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>Default save path for downloads</string>
       </property>
       <widget class="QLineEdit" name="downloadPath">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>22</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="browseButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>20</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </widget>
      <widget class="QCheckBox" name="showCheckboxes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show checkboxes for batch operations</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="drawMetrics">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Draw metric grid on sliders</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="spoilOnActivate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Spoil item on activation</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>100</y>
         <width>321</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Item info (bitrate, duration, etc...)</string>
       </property>
       <widget class="QCheckBox" name="showInfo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>311</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>321</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;Very slow for remote files. Require restart of program&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tabber_tab">
      <attribute name="title">
       <string>Playlist</string>
      </attribute>
      <widget class="QComboBox" name="tabPositionSelect">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Tab position</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Item font</string>
       </property>
      </widget>
      <widget class="QComboBox" name="itemFontSelect">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="itemFontSize">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>70</y>
         <width>34</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="itemInfoFontSize">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>140</y>
         <width>34</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Item info font</string>
       </property>
      </widget>
      <widget class="QComboBox" name="itemInfoFontSelect">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Default item 
text color</string>
       </property>
      </widget>
      <widget class="QPushButton" name="defaultItemTextColorButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Selected item 
text color</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selectedItemTextColorButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>100</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Default item info 
 text color</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selectedItemInfoTextColorButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>170</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>170</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Selected item info 
 text color</string>
       </property>
      </widget>
      <widget class="QPushButton" name="defaultItemInfoTextColorButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QSpinBox" name="itemHeightSize">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>18</number>
       </property>
       <property name="value">
        <number>18</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Item height</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="scrollButtonsCheck">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Use scroll buttons on tabs</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="useSystemIconsCheck">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Use system icons for elems</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="hot_keys_tab">
      <attribute name="title">
       <string>Hot keys</string>
      </attribute>
      <widget class="QTreeView" name="treeView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>374</width>
         <height>296</height>
        </rect>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="colors_tab">
      <attribute name="title">
       <string>Item Colors</string>
      </attribute>
      <widget class="QPushButton" name="defaultColorButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Default color</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Listened color</string>
       </property>
      </widget>
      <widget class="QPushButton" name="listenedColorButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Liked color</string>
       </property>
      </widget>
      <widget class="QPushButton" name="likedColorButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Played color</string>
       </property>
      </widget>
      <widget class="QPushButton" name="playedColorButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Folder color</string>
       </property>
      </widget>
      <widget class="QPushButton" name="folderColorButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="useGradientCheck">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Use gradient</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Spectrum</string>
      </attribute>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Spectrum update frequency</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spectrumUpdateFrequecy">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>18</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Spectrum bars count</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spectrumBarsCount">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>512</number>
       </property>
       <property name="value">
        <number>18</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="spectrumMonocolorUse">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>231</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Use custom colors</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Custom color 1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="spectrumColor">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>millis</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spectrumHeight">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>40</number>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Spectrum height</string>
       </property>
      </widget>
      <widget class="QPushButton" name="spectrumColor2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>180</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Custom color 2</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="spectrumComboUse">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>181</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Combine channels to one layer</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spectrumMultiplier">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="value">
        <number>9</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Signal multiplier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="spectrumColor3">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>200</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>180</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Custom color 3</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
